go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
+	github.com/go-logr/logr v0.3.0
```

Comment:
Do we need this?

Reply:
explanation by @JoelSpeed: "This interface is what controller runtime uses, so if we just use klog directly, we lose the logging that controller runtime would emit, so it's better to use the logger interface so we are sharing what controller runtime is doing"

Reply:
This is already a dependency, though just be an indirect dependency up to this point, looks like we have already run `make vendor` on this PR right? As there are no changes to go.sum this shows it's not a new depdencency

Reply:
Yes, seems thats the case.